src/generate-font-glyph-enums.ts

```diff
@@ -2,6 +2,20 @@ import fonts from '@hayes0724/web-font-converter/src/lib/fonts.js';
 import { loadSync } from 'opentype.js';
 import { writeFile } from './helpers.js';
 
+import { createHash } from 'crypto';
+import { readFileSync } from 'fs';
+
+const currentHash = readFileSync('./data/font.hash').toString().trimEnd();
+
+const fileBuffer = readFileSync('./Destiny-2-Font-Symbols/fonts/Destiny_Keys.otf');
+const hexHashSum = createHash('sha1').update(fileBuffer).digest('hex');
+
+if (currentHash === hexHashSum) {
+  process.exit();
+}
+
+writeFile('./data/font.hash', hexHashSum);
```

Comment:
Should probably write this after running the converter successfully so that an error in the converter won't result in it skipping next time.